lib/fastlane/plugin/ddg_apple_automation/actions/update_asana_for_release_action.rb

```diff
@@ -56,6 +57,10 @@ def self.available_options
           FastlaneCore::ConfigItem.github_token,
           FastlaneCore::ConfigItem.is_scheduled_release,
           FastlaneCore::ConfigItem.platform,
+          FastlaneCore::ConfigItem.new(key: :tag,
```

Comment:
If you add `env_name: "TAG"` to this definition, it won't need any changes in the GHA workflow file 👍

Reply:
Actually scratch that. Better be explicit about parameters and update the workflow.